knowledge_base/job_read_secret/databricks.yml

```
@@ -0,0 +1,47 @@
+bundle:
+  name: job-read-secret-example
+
+# workspace:
+#   host: https://myworkspace.cloud.databricks.com
+
+resources:
+  secret_scopes:
+    my_secret_scope:
+      name: secrets-scope-1
+      permissions:
+        - level: CAN_VIEW
+          group_name: users
+        - level: CAN_MANAGE
+          group_name: admins
+  jobs:
+    example_python_job:
+      name: "example-python-job"
+      parameters:
+        - name: "scope_name"
+          default: ${resources.secret_scopes.my_secret_scope.name}
+      tasks:
+        - task_key: example_python_task
+          spark_python_task:
+            python_file: "src/example_spark_python_task.py"
+            parameters:
+              - --scope_name={{job.parameters.scope_name}}
+
+# Defines the targets for this bundle.
+# Targets allow you to deploy the same bundle to different Databricks workspaces.
+targets:
+  prod: {
+    # No overrides
+  }
```

```suggestion
  prod: 
    # No overrides
  
```

nit: can omit brackets.

---

i got the braces back - otherwise the syntax highlighter in vscode is not happy